Source/WebInspectorUI/UserInterface/Views/ColorPicker.js

```diff
@@ -171,6 +211,24 @@ WI.ColorPicker = class ColorPicker extends WI.Object
         });
     }
 
+    // Static
+    static accessabilityContrast(color1, color2)
+    {
+        function getLuminance(color)
+        {
+            let RsRGB = color[0] / 255;
+            let GsRGB = color[1] / 255;
+            let BsRGB = color[2] / 255;
+            let R = RsRGB <= 0.03928 ? RsRGB / 12.92 : ((RsRGB + 0.055) / 1.055) ** 2.4;
+            let G = GsRGB <= 0.03928 ? GsRGB / 12.92 : ((GsRGB + 0.055) / 1.055) ** 2.4;
+            let B = BsRGB <= 0.03928 ? BsRGB / 12.92 : ((BsRGB + 0.055) / 1.055) ** 2.4;
+            return 0.2126 * R + 0.7152 * G + 0.0722 * B;
+        }
+        let L1 = getLuminance(color1.rgb);
+        let L2 = getLuminance(color2.rgb);
+        return (Math.max(L1, L2) + 0.05) / (Math.min(L1, L2) + 0.05);
```

Comment:
Might be good to check with CSS WG and WCAG members of the WebKit team on whether exposing the WCAG 2.1 algorithm makes sense. Relatively recently, the decision was made not to ship the color-contrast CSS function due to the use of this algorithm, and the WG has made it clear they think it is problematic (https://drafts.csswg.org/css-color-6/#issue-b57775e7)

If we do want to keep this, it would be better if we could avoid multiple implementations of this algorithms, and instead call into WebCore's implementation.